spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/http/DefaultEurekaClientHttpRequestFactorySupplier.java

```diff
@@ -45,7 +48,10 @@
  * @author Jiwon Jeon
  * @since 3.0.0
  */
-public class DefaultEurekaClientHttpRequestFactorySupplier implements EurekaClientHttpRequestFactorySupplier {
+public class DefaultEurekaClientHttpRequestFactorySupplier
```

Comment:
Please add full name and surname in the javadocs of any changed and added classes.